src/Blazored.Typeahead/BlazoredTypeahead.razor.cs

```diff
@@ -1,120 +1,121 @@
+
+        [Parameter] public TValue Value { get; set; }
+        [Parameter] public EventCallback<TValue> ValueChanged { get; set; }
+        [Parameter] public Expression<Func<TValue>> ValueExpression { get; set; }
+
         [Parameter] public IList<TValue> Values { get; set; }
         [Parameter] public EventCallback<IList<TValue>> ValuesChanged { get; set; }
-        [Parameter] public Expression<Func<IList<TValue>>> ValuesExpression { get; set; }
-
-        [Parameter] public Func<string, Task<IEnumerable<TItem>>> SearchMethod { get; set; }
+        [Parameter] public Expression<Func<IList<TValue>>> ValuesExpression { get; set; }
+
+        [Parameter] public Func<string, Task<IEnumerable<TItem>>> SearchMethod { get; set; }
         [Parameter] public Func<TItem, TValue> ConvertMethod { get; set; }
 
-        [Parameter] public RenderFragment NotFoundTemplate { get; set; }
-        [Parameter] public RenderFragment<TItem> ResultTemplate { get; set; }
-        [Parameter] public RenderFragment<TValue> SelectedTemplate { get; set; }
-        [Parameter] public RenderFragment HeaderTemplate { get; set; }
-        [Parameter] public RenderFragment FooterTemplate { get; set; }
-
-        [Parameter(CaptureUnmatchedValues = true)] public IReadOnlyDictionary<string, object> AdditionalAttributes { get; set; }
-        [Parameter] public int MinimumLength { get; set; } = 1;
-        [Parameter] public int Debounce { get; set; } = 300;
-        [Parameter] public int MaximumSuggestions { get; set; } = 10;
-        [Parameter] public bool Disabled { get; set; } = false;
-        [Parameter] public bool EnableDropDown { get; set; } = false;
-        [Parameter] public bool ShowDropDownOnFocus { get; set; } = false;
-
-        protected bool IsSearching { get; private set; } = false;
-        protected bool IsShowingSuggestions { get; private set; } = false;
-        protected bool IsShowingMask { get; private set; } = false;
+        [Parameter] public RenderFragment NotFoundTemplate { get; set; }
+        [Parameter] public RenderFragment<TItem> ResultTemplate { get; set; }
+        [Parameter] public RenderFragment<TValue> SelectedTemplate { get; set; }
+        [Parameter] public RenderFragment HeaderTemplate { get; set; }
+        [Parameter] public RenderFragment FooterTemplate { get; set; }
+
+        [Parameter(CaptureUnmatchedValues = true)] public IReadOnlyDictionary<string, object> AdditionalAttributes { get; set; }
+        [Parameter] public int MinimumLength { get; set; } = 1;
+        [Parameter] public int Debounce { get; set; } = 300;
+        [Parameter] public int MaximumSuggestions { get; set; } = 10;
+        [Parameter] public bool Disabled { get; set; } = false;
+        [Parameter] public bool EnableDropDown { get; set; } = false;
+        [Parameter] public bool ShowDropDownOnFocus { get; set; } = false;
+        [Parameter] public string Class { get; set; } = "";

```

Comment:
Here's the actual change. I hate that the diff sucks!

Reply:
I'll push a fix for these whitespace changes.

Reply:
Whew, got it, thanks to here:

https://til.hashrocket.com/posts/696df00135-remove-whitespace-changes-then-git-add-p